content/blog/2020-07-16-devops-for-data-scientists.md

```diff
@@ -258,7 +258,7 @@ on: [push]
 jobs:
   run:
     runs-on: [ubuntu-latest]
-    container: docker://dvcorg/cml-py3:latest
+    container: iterativeai/cml:0-dvc2-base1
```

Comment:
does this image have python 3?

Reply:
Yes, all the images have Python 3; please take a look to the [documentation](https://cml.dev/doc/self-hosted-runners#docker-images).

Reply:
true is base1 I always get confused with cml:0